config/routes.rb

```diff
@@ -1,6 +1,11 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
+  resources :users, only: [:index, :new, :create, :udpdate, :show] do
+    resources :posts, only: [:index, :new, :create, :show] do
+      resources :comments, only: [:index, :new, :create, :show] 
+      resources :likes, only: [:create, :destroy]
+    end
+  end
```

Comment:
- [ ] Also for the routes, We don't need to expose unnecessary routes. we only need `:index, :show` routes for both the `user` and the `post` controller. Please consider updating your routes.

Reply:
Hi @TadesseDev. Thank you for your review! Can I ask how am i going to create a new post if i don't have the other actions defined, pls?

Reply:
Dear @Zven94, at this point creating a new post or user is not part of the requirement. There will be a time to do that and you will be asked. For now, please follow the requirements and creat only the necessary route and actions,